### ISSUE TITLE
Incompatible with Windows 10

### ISSUE THREAD

**Issue by stefanrbk** [koalaman/shellcheck#763]
cabal is currently incompatible with the Linux subsystem for Windows 10 due to the timer_create syscall, a requirement for ghc, not being implemented yet by Microsoft.

**Comment by morsik** [comment]
Shouldn't this bug be reported to Microsoft then? ;)

**Comment by stefanrbk** [comment]
Yes, it is a Microsoft issue, but I felt a warning on your side might not
be a bad idea

**Comment by koalaman** [comment]
ShellCheck can actually be compiled natively for Windows, you don't need the Linux subsystem.

**Comment by rogalmic** [comment]
This "timer_create" issue is fixed in recent insider's build - it will probably go public in creators update (probably April 2017). I have not tried native shellcheck.

I was able to use slightly modified shellcheck plugin in Visual Studio Code on Win 10:

![image](https://cloud.githubusercontent.com/assets/23117003/23165704/c4068820-f83d-11e6-898f-52f8b54d0aac.png)

![image](https://cloud.githubusercontent.com/assets/23117003/23165921/b70b35e8-f83e-11e6-9418-c4b886f01af1.png)


Code change +plugin mod:
https://github.com/timonwong/vscode-shellcheck/issues/3

**Comment by koalaman** [comment]
That's pretty awesome ^____^
